pkg/defaults/defaults.go

```diff
@@ -92,6 +92,8 @@ const (
 	DefaultInfoRedisPrefix       = "INFO_EVE_"
 	DefaultMetricsRedisPrefix    = "METRICS_EVE_"
 	DefaultRequestsRedisPrefix   = "REQUESTS_EVE_"
+	DefaultEveLogLevel           = "info"
```

Comment:
Is it the most informative one?

Reply:
No, but it is defaults from here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md